framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -89,15 +90,15 @@ export const CreatableSelectDropdown = <T extends string = string> ({
   onOptionChange,
   width = '100%',
   variant = 'outline',
-  initialValue,
```

Comment:
we still want to support `initialValue`.
Check if it's implemented in `<Select />`

Reply:
I think it's called defaultValue for select

Reply:
So tru!